tests_config/run_spec.js

```diff
@@ -22,12 +22,13 @@ global.run_spec = (dirname, parsers, options) => {
     throw new Error(`No parsers were specified for ${dirname}`);
   }
 
-  fs.readdirSync(dirname).forEach(basename => {
+  fs.readdirSync(dirname, { withFileTypes: true }).forEach(dirent => {
```

Comment:
This part can commit to `next` branch, already requires node 10. maybe also change to `for...of`

Reply:
fixed it. Thank you.